universal-application-tool-0.0.1/app/controllers/dev/DatabaseSeedController.java

```diff
@@ -259,6 +259,7 @@ private void truncateTables() {
         Account.class,
         Applicant.class,
         Application.class,
+        models.Version.class,
```

Comment:
Looks like you need at least one active version.

Reply:
Ah you're right - this didn't impact the tests I was running on staging.  🤦